src/main/java/org/ibp/api/java/location/LocationService.java

```diff
@@ -13,6 +14,9 @@
 
 	List<LocationTypeDTO> getLocationTypes();
 
+	long countLocations(String crop, String programUUID, Set<Integer> locationTypes, final List<Integer> locationIds,
```

Comment:
Refactor `countLocations `and `getLocations `should use `LocationSearchRequest `DTO